dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/PavedRoadFramework.kt

```diff
@@ -69,7 +69,8 @@ abstract class PavedRoadFramework(
     open fun getComponentFactoriesForIntermediateRepresentation(
         context: ApplicationContext,
     ): List<TemplateComponentFactory> {
-        return context.getBeansOfType<TemplateComponentFactory>().values.toList()
+        val containerBean = context.getBean<ComponentFactoryContainer>()
+        return containerBean.factories.reversed()
```

Comment:
Why reversed? Where is the order controlled before this?

Reply:
To get the warning factory to the end I guess? The rest of the order is somewhat arbitrary?

Reply:
But why was the warning added for this framework as well.